sygnal/apnspushkin.py

```diff
@@ -131,8 +132,18 @@ def __init__(self, name, sygnal, config):
             if not self.get_config("topic"):
                 raise PushkinSetupException("You must supply topic.")
 
+        # use the Sygnal global proxy configuration
+        proxy_url_str = sygnal.config["proxy"].get("url")
```

Comment:
won't this fail on existing configs with no `proxy` section?

Reply:
No, the defaults take care of that